pkg/operator/ceph/csi/template/rbd/csi-rbdplugin.yaml

```diff
@@ -123,6 +123,11 @@ spec:
               mountPath: /etc/ceph/ceph.conf
               subPath: ceph.conf
             {{ end }}
+            {{ if .EnableOIDCTokenProjection }}
+            - name: oidc-token
+              mountPath: /run/secrets/tokens
```

Comment:
Don't we need the new version of ceph-csi that includes the fix for this path?

Reply:
> Don't we need the new version of ceph-csi that includes the fix for this path?

This will only be used in cephcsi v3.6 which is not out yet.
But this change harmless, even with v3.5